cli/dcoscli/main.py

```diff
@@ -89,43 +91,57 @@ def _main():
     subproc = Popen([executable,  command] + args['<args>'],
                     stderr=PIPE)
 
+    if log_level:
```

Comment:
env vars are only set for the current process.  they don't persist in the parent process.